### ISSUE TITLE
Swipe gesture for non-trackpad device

### ISSUE THREAD

**Issue by koenpunt** [nylas/N1#1436]
For my Mac I use a Wacom tablet, and it would be nice if the swipe gestures which are now available for the trackpad, would also be supported for a tablet (or even a regular mouse) device.

I understand this has some implications, mainly in respect to dragging messages. But as I'm (almost) never dragging messages, it would like to have this behavior as an option.

**Comment by bengotow** [comment]
Hi @koenpunt — is there any way to make your Wacom tablet emit "touch" events? That'd allow you to use it for gestures.

It looks like Wacom has their own touch SDK: http://www.wacomeng.com/touch/WacomFeelMulti-TouchFAQ.htm, but it's unclear whether there's any way to make their default driver act as a touch surface rather than a pointer device.

**Comment by koenpunt** [comment]
My wacom doesn't support touch, but I like to make the gesture with my pen. A combination of handling javascript `mousedown` and `mousemove` should work I guess?

**Comment by bengotow** [comment]
Yep—I think that's what we'd need to do. We currently support `touchstart` and `touchend` which is why I was hoping it could emit those events. If that's not possible we'll need to use the mouse events and make an option to enable swiping via mouse.

**Comment by codyhatch** [comment]
I would **strongly** like an option to enable swiping via mouse.  Mailbox supported this, and I loved it for quickly managing emails with a regular desktop mouse.

I realize this might interfere with drag and drop, but I never drag and drop emails, and I love swiping.  It's a tradeoff I'd like the option to make.  Honestly, it's the only thing I miss from Mailbox.

**Comment by koenpunt** [comment]
Btw, @bengotow you labeled it `platform-os-x`, but I think this also applies to windows and linux

**Comment by todesignandconquer** [comment]
I echo @codyhatch's sentiment and would love being able to swipe by clicking and dragging because while swiping on a Magic Mouse works, it's very unintuitive.
